celery/backends/redis.py

```diff
@@ -340,3 +341,22 @@ def db(self):
     @deprecated.Property(4.0, 5.0)
     def password(self):
         return self.connparams['password']
+
+
+class SentinelBackend(RedisBackend):
+    """Sentinel task result store."""
+
+    def __init__(self, *args, **kwargs):
+        super(SentinelBackend, self).__init__(*args, **kwargs)
+
+        _get = self.app.conf.get
+
+        self.broker_url = _get('BROKER_URL') or {}
```

Comment:
This code demonstrates an approach that is based on the following principles:
- it is necessary to use a a child class that inherits from the RadishBakend;
- tt is necessary to use a kombu.Connection, because the whole backend uses kombu.